src/Database/Schema/PostgresSchema.php

```diff
@@ -340,6 +345,7 @@ public function columnSql(Table $table, $name)
             'datetime' => ' TIMESTAMP',
             'timestamp' => ' TIMESTAMP',
             'uuid' => ' UUID',
+            'json' => ' JSON'
```

Comment:
Should we do a version check in postgres too? JSON types were only added in 9.2.